rudder-agent/SOURCES/rudder.init

```diff
@@ -47,6 +47,11 @@ get_service_name()
     SERVICE_NAME=$(ls -1 /etc/init.d | grep "${1}" | tail -n 1)
   fi
 
+  # If we can not get the service name, fallback to the user-supplied name
```

Comment:
This prevents the case where a script / service can not be found and returns an empty echo. We should at lease give back what the user proposed.